### PR TITLE
Feature/39434

### PR DESCRIPTION
**Pull request by kjugi**
DEV-39434

Add improvements in styles in the component product page 

+ Adjust styles from swatches
+ Adjust styles from rating
+ Add new and adjust product-page styles
+ Update additional-changes in product-page template
+ Add styles to the gallery and adjust it to fotorama
+ Add new table variant to the product page

### CODE REVIEW

File: components/02-elements/rating/_rating.scss [hunk 1]
```diff
@@ -13,16 +16,67 @@ $icon__inactive-color           : '#c2c1df' !default;
...
         justify-content: center;
         align-items: center;
-        direction: rtl;
     }
+
+    &--review {
+        width: $rating__size--review;
+        margin-bottom: $rating__margin--review;
+
+        & .rating__rate-item {
```
Comment (szafran89) [inline components/02-elements/rating/_rating.scss]: please remove ampersand

Comment (kjugi) [reply components/02-elements/rating/_rating.scss]: removed

File: components/03-modules/gallery/_gallery.scss [hunk 2]
```diff
@@ -114,4 +114,19 @@ $gallery__thumb-width--large               : 80px !default;
         left: $gallery__label-offset-left;
         top: $gallery__label-offset-top;
     }
+
+    &-placeholder {
+        position: relative;
+
+        & .loader {
```
Comment (szafran89) [inline components/03-modules/gallery/_gallery.scss]: please remove umpersand

File: components/03-modules/gallery/_gallery.scss [hunk 3]
```diff
@@ -86,11 +86,11 @@ $gallery__thumb-width--large               : 80px !default;
...
+        display: flex !important;
+        justify-content: center;
+        align-items: center;
         width: $gallery__thumb-arrow-width;
         height: $gallery__thumb-arrow-height;
-        display: flex;
-        justify-content: center;
-        align-content: center;
         margin: 0 auto;
         &:hover {
```
Comment (szafran89) [inline components/03-modules/gallery/_gallery.scss]: missing focus

File: components/03-modules/quantity-update/_quantity-update.scss [hunk 4]
```diff
@@ -21,10 +21,12 @@ $quantity-updater__icon-spacing                      : 12px !default;
         border-bottom: $quantity-update__input-border;
         border-left: 0;
         border-right: 0;
+        border-radius: 0;
         width: $quantity-update__input-width;
         height: $quantity-update__input-width;
         color: $quantity-update__input-color;
         text-align: center;
+        -webkit-appearance: none;
```
Comment (szafran89) [inline components/03-modules/quantity-update/_quantity-update.scss]: is it not enough to add `appearance: none;` ?

File: components/03-modules/gallery/_gallery.scss (hunk 2 above)
Comment (kjugi) [reply components/03-modules/gallery/_gallery.scss]: removed

File: components/03-modules/gallery/_gallery.scss (hunk 3 above)
Comment (kjugi) [reply components/03-modules/gallery/_gallery.scss]: thanks, added in one more place

File: components/03-modules/quantity-update/_quantity-update.scss (hunk 4 above)
Comment (kjugi) [reply components/03-modules/quantity-update/_quantity-update.scss]: working same, changed and pushed

**Pull request merged**